test/packages/other/skipped_tests/data_stream/skip/_dev/test/system/test-default-config.yml

```diff
@@ -0,0 +1,8 @@
+skip:
+  reason: There are undefined fields.
+  link: https://github.com/elastic/elastic-package/issues/1
```

Comment:
Message shown when running system tests in this package:
https://buildkite.com/elastic/elastic-package/builds/3400#01900e89-6d80-4bcb-ac1b-6c57fd9e5f0b/216-2243
```
2024/06/12 22:30:52  WARN skipping system test for skipped_tests/skip: There are undefined fields. (details: https://github.com/elastic/elastic-package/issues/1)
```

Test results file:
```xml
<testsuites>
  <testsuite name="system" tests="1" skipped="1">
    <!-- test suite for system tests -->
    <testcase name="system test: default" classname="skipped_tests.skip" time="2.4222e-05">
      <skipped message="There are undefined fields. [https://github.com/elastic/elastic-package/issues/1]"/>
    </testcase>
  </testsuite>
</testsuites>
```